memory.md

```diff
@@ -35,8 +35,11 @@ Example: Configuring 1GB initial on-heap, 2GB max on-heap, 8GB off-heap:
 
 ```-Xms1G -Xmx2G -Dorg.bytedeco.javacpp.maxbytes=8G -Dorg.bytedeco.javacpp.maxphysicalbytes=8G```
 
+**Notice**: With GPU systems, the maxbytes and maxphysicalbytes settings currently also effectively defines the memory limit for the GPU, since the off-heap memory is mapped (via NDArrays) to the GPU - read more about this in the GPU-section below.
+
```

Comment:
Maybe we might want to add a link to: http://bytedeco.org/javacpp/apidocs/org/bytedeco/javacpp/Pointer.html#maxPhysicalBytes  http://bytedeco.org/javacpp/apidocs/org/bytedeco/javacpp/Pointer.html#maxBytes

Another thing mentioned was clarifying how gpu memory was allocated..the problem I had was how to express this without going down a rabbit hole of how gpu memory works.

Reply:
There is evidently a difference between those two values, though. In my current setting, I want a large heap, so I used 
-Xms8G -Xmx16G -Dorg.bytedeco.javacpp.maxbytes=8G -Dorg.bytedeco.javacpp.maxphysicalbytes=8G
This resulting in that dl4j wouldn't even "boot" (crashed on this line: Nd4j.getMemoryManager().setAutoGcWindow(10000) in my code)
The reason was that physical size (total of JVM) at init point was 10GB, while "maxphysicalbytes" was 8GB.

This at least made the thing run:
-Xms8G -Xmx14G -Dorg.bytedeco.javacpp.maxbytes=8G -Dorg.bytedeco.javacpp.maxphysicalbytes=16G

Reply:
Yes, physical memory usage of the whole process includes JVM heap. Please add that clarification as well since that would have helped you! Thanks